### DISCUSSION TITLE
How to keep script updated?

### DISCUSSION THREAD

**Discussion by elisimpson** [007revad/Synology_HDD_db#178] [Q&A] [answered]
I have the script scheduled to run on boot up, but of course that doesn't happen very often. Is there a switch or something I could schedule to run more often that only updates the script if there's a new version?

**Comment by 007revad** [comment, marked as answer]
You can use the autoupdate update option.
```
--autoupdate=AGE
    AGE is how many days old a release must be before auto-updating. 
    AGE must be a number: 0 or greater.
```

I'd recommend setting the age to 3 days or more, because I do occasionally make a mistake that I may not notice right away.

When your Synology reboots and runs the script with the --autoupdate option it will: 
1. Check for an updated version
2. Automatically download the new version.
3. Run the new script version.

If there is no Internet access, after 30 seconds the script will continue running the current version.
